Client/Frontend/Reader/ReaderModeStyleViewController.swift

```diff
@@ -5,78 +5,123 @@
 import Foundation
 import UIKit
 
+private struct ReaderModeStyleViewControllerUX {
+    static let RowHeight = 50
+
+    static let Width = 270
+    static let Height = 4 * RowHeight
```

Comment:
I could not get this to work with just constraints. Even with all constraints in place I have to set `UIViewController.preferredContentSize` to get the right popover size.